src/OmnichannelChatSDK.ts

```diff
@@ -474,10 +480,16 @@ class OmnichannelChatSDK {
                 pollingInterval: 30000
             };
 
+            const tokenRefresher = async (): Promise<string> => {
+                await this.getChatToken(false);
```

Comment:
Is there error handling for this in chat sdk?

Reply:
Yes, the failure would be logged in telemetry if any. In this situation, the old token would still be used then messaging would eventually fail.